views/proxy_confirmation.pug

```diff
@@ -2,17 +2,22 @@ extends layout
 
 block content
   section.divider#request-info
-    h3= _("Oauth request for %s from: %s").format(kind, redirect_address)
+    h2= _("Oauth request")
+    span= _(" ")
+    h3= _("for %s from: %s").format(kind, redirect_address)
+
     div.panel-body
           p.text-center
-            = _('Are you sure you would like to grant %s access to the credentials of device: %s').format(redirect_address, kind)
+            = _('Are you sure you would like to grant #[strong %s] access to the credentials of device: #[strong %s]').format(redirect_address, kind)
```

Comment:
This doesn't work. Pug syntax is not interpreted inside a JS expression.

Reply:
ok